assets/scripts/embedding_tests_dev_0.0.1.min.html

```diff
@@ -8,21 +8,27 @@ <h2>v0.0.1 (minified)</h2>
 
 <script src="/assets/scripts/oppia-player-0.0.1.js"></script>
 <script src="/assets/scripts/embedding_tests.js"></script>
+<script src="../../third_party/static/jquery-3.2.1/jquery.min.js"></script>
+
+<form action="#" onsubmit="update()">
+  <input id="embedding-exploration-id" type="text" placeholder="explorationId">
+  <input id="embedding-submit-btn" type="submit" value="Submit">
```

Comment:
Give more scaffolding: have the value be something like "Update Exploration ID"

Reply:
done.